src/moments.jl

```diff
@@ -160,39 +270,39 @@ function _moment3(v::RealArray, m::Real)
         @inbounds z = v[i] - m
         s += z * z * z
     end
-    s / n
+    varcorrection(n, false) * s
```

Comment:
That's why I've set it to `corrected=false` which is equivalent to `s / n` or `(s / sum(w))`. If you'd prefer I can keep the `varcorrection` usage to just `_moment2`.